psi4/src/psi4/libtrans/integraltransform_moinfo.cc

```diff
@@ -201,7 +201,7 @@ void IntegralTransform::process_spaces() {
             for (int h = 0; h < nirreps_; ++h) {
                 for (int n = 0; n < aOrbsPI[h]; ++n) {
                     if (transformationType_ == TransformationType::Restricted) {
-                        aPitzerCount = pitzerOffset + nalphapi_[h];
+                        aPitzerCount = pitzerOffset + nbetapi_[h];
```

Comment:
@andysim Could you explain this one? Intuitively I would have guess alpha as well.

Reply:
I think beta is correct here.  At some point @lothian modified [this line](https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libtrans/integraltransform_moinfo.cc#L190) to correctly include `SOCC` orbitals in the `VIR` space.  In ROHF `SOCC` orbitals appear as both `OCC` and `VIR`, with the appropriate denominators set to a very large value to prevent double counting.  The change from TDC on line 190 is the correct way to do this, and my modification of line 204 makes the offsets consistent, removing the memory error.  The error we're seeing in the test now is that the SCF solver is reading in the excited state guess, but then falling back to the ground state.  For once our solver is too good!  Some kind of MOM should help here - I'll see what I can do.